### DISCUSSION TITLE
[Config Support]: disable review bar from live view?

### DISCUSSION THREAD

**Discussion by zenguru84** [blakeblackshear/frigate#15031] [Config Support]
### Describe the problem you are having

is there a way to disable the review bar from top of live view page?
for most of the time i get reviews of me an my family moving around in the yard, so not important.
the thing is i use the live view on a screen and that bar pushes down the live streams. the bottom ones becoming only partially visible.

it will also be great to have the button "mark these items as reviewed" somewhere up in the review page, to avoid scrolling to the bottom of days of items (with me walking around) just to mark them as reviewed.

i'll open a feature request, but maybe these are doable it's just me not knowing how to do it. :) 

### Version

0.14.1

### Frigate config file

```yaml
-
```

### Relevant Frigate log output

```shell
-
```

### Relevant go2rtc log output

```shell
-
```

### Operating system

Proxmox

### Install method

Proxmox via Docker

### docker-compose file or Docker CLI command

```yaml
-
```

### Object Detector

Coral

**Comment by hawkeye217** [comment]
No, it's not possible to remove the "filmstrip" from the Live dashboard. However, only review items classified as Alerts will appear there. See the [review docs](https://docs.frigate.video/configuration/review/) on how to classify items as detections instead of alerts. 

On the Review page, you can use the keyboard shortcut Ctrl/Cmd-A to select all and then use the "Mark as reviewed" button in the top bar to quickly mark all your review items as reviewed.
